### PR TITLE
Add meeting minutes for SVSM Development Call on October 23rd, 2024

### PR DESCRIPTION
**Pull request by joergroedel**

### CODE REVIEW

File: Meetings/devel-call-2024-10-23.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+## Attendees:
+
+Adam Dunlap, Chris Hawblitzel, Cláudio Carvalho, Dionna Glaze, Huibo Wang, James Bottomley, Joerg Roedel, Jon Lange, Mine & John Starks, Nicolai Stange, Oliver Steffen, Peter Fang, Roy Hopkins, Stefano Garzarella, Tom Dohrmann, Tom Lendacky, Vasant Karasulli, Vijay Dhanraj, Weidong Cui, Ziqiao Zhou
+
+## Topics:
+
+### Presentation and Discussion about formal verification with Verus (from Microsoft Research)
+
+* Goal: To prove the security and correctness of the COCONUT-SVSM code using formal verification.
+* Verus: A formal verification tool for Rust developed by Microsoft, CMU, and VMware ([Slides](Data/verismo-10-23-2024-talk.pdf)).
```
Comment (osteffenrh) [inline Meetings/devel-call-2024-10-23.md:12]: missing file?

Comment (osteffenrh) [reply Meetings/devel-call-2024-10-23.md:12]: Sorry, I see it now.

**Review by osteffenrh — APPROVED** [review]

**Comment by ziqiaozhou** [comment]
Thanks for adding the meeting note. I just noticed that the slides I shared with you has a small typo in it. I revised it and it would be better if you can use the revised one as a public material. [verismo-10-23-2024-talk.pdf](https://github.com/user-attachments/files/17511380/verismo-10-23-2024-talk.pdf)

**Comment by joergroedel** [comment]
> Thanks for adding the meeting note. I just noticed that the slides I shared with you has a small typo in it. I revised it and it would be better if you can use the revised one as a public material. [verismo-10-23-2024-talk.pdf](https://github.com/user-attachments/files/17511380/verismo-10-23-2024-talk.pdf)

Thanks, I updated the PR with the new PDF.

**Pull request merged**